workflows/phageannotator/main.nf

```diff
@@ -71,39 +42,22 @@ def multiqc_report = []
 
 workflow PHAGEANNOTATOR {
 
+    take:
+    fastq_gz
+    fasta_gz
```

Comment:
```suggestion
    fasta_gz  // [ [ meta ], fasta.gz ]   , assemblies/genomes (mandatory)
```

Reply:
recommend adding info about input about how the channel should like.....above is just an ex